File: core/src/main/java/uk/lobsterdoodle/namepicker/Logger.java
package uk.lobsterdoodle.namepicker;

public interface Logger {
    void v(String tag, String msg);

    void d(String tag, String msg);

    void i(String tag, String msg);

    void w(String tag, String msg);

    void e(String tag, String msg, Throwable ex);

    void wtf(String tag, String msg, Throwable ex);
}

File: core/src/main/kotlin/uk/lobsterdoodle/namepicker/Logger.kt
package uk.lobsterdoodle.namepicker

interface Logger {
    fun v(tag: String, msg: String)
    fun d(tag: String, msg: String)
    fun i(tag: String, msg: String)
    fun w(tag: String, msg: String)
    fun e(tag: String, msg: String, ex: Throwable)
    fun wtf(tag: String, msg: String, ex: Throwable)
}
